packaging/resource_suite_s3_nocache.py

```diff
@@ -1258,6 +1258,40 @@ def test_simultaneous_open_writes(self):
                 os.unlink(rule_file_path)
 
 
+    def test_read_write_zero_length_file_issue_1890(self):
+
+        rule_file_path = 'test_issue_1890.r'
+
+        rule_str = '''
+test_open_write_close {
+    msiDataObjCreate("/tempZone/home/otherrods/file1.txt", "destRescName=demoResc", *fd)
```

Comment:
could replace these hard-coded strings with constructed-from-variables... (otherrods)

Reply:
Done in the latest push.  This is in [#1897].

Reply:
The test case change is [#1897].